build.gradle

```diff
@@ -1,8 +1,8 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
 
-  ext.KOTLIN_VERSION = "1.5.0"
-  ext.ANDROID_PLUGIN_VERSION = '4.2.0'
+  ext.KOTLIN_VERSION = "1.4.32"
+  ext.ANDROID_PLUGIN_VERSION = '7.0.0-alpha14'
```

Comment:
beta 1 should be out now? can change later too

Reply:
beta isn't out yet, upgraded to alpha15